docs/DeveloperGuide.md

```diff
@@ -651,33 +716,39 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     Steps 1a1-1a3 are repeated until the data entered are correct. <br>
     Use case resumes from step 2.
 
-**Use case: UC03 Add a new product to supplier**
+**Use case: UC03 Assign a macro**
 
 **MSS**
 
-1. User keys in command to add a product.
-2. CLI-nic adds the product into the list and shows a success message.
+1. User keys in command to assign a macro with an alias and a command string to be saved.
+2. CLI-nic saves the macro and shows a success message.
 
     Use case ends.
 
 **Extensions**
 
-* 1a. CLI-nic detects an overlap in the entered product and the existing products in the supplier's supply list.
+* 1a. The alias or command string supplied is invalid.
 
-  * 1a1. CLI-nic shows an error message.
-  * 1a2. CLI-nic requests for a new product name.
-  * 1a3. User enters a product name.
+  * 1a1. CLI-nic shows an error message with the constraints for the fields required.
+  * 1a2. User enters a new command.
 
-    Steps 1a1-1a3 are repeated until the name entered is correct. <br>
+    Steps 1a1-1a2 are repeated until the data entered is valid. <br>
```

Comment:
Should we standardise whether to use correct or valid?

For data and format

Reply:
Not sure which is better, both should be fine since the exact type of data is shown in the extension description.